authentik/events/models.py

```diff
@@ -305,6 +305,16 @@ def __str__(self) -> str:
     class Meta:
         verbose_name = _("Event")
         verbose_name_plural = _("Events")
+        indexes = [
+            models.Index(fields=["action"]),
+            models.Index(fields=["user"]),
```

Comment:
Would user__username be enough?

Reply:
I thought about that too, but we might want to filter on user.pk too so might as well index the entire field

Reply:
Oh yeah I didn't notice that it was only pk, username and email in there.